scrapers/base.py

```diff
@@ -62,14 +62,12 @@ def is_container_build(self, build_info):
         except (ValueError, TypeError):
             extra_json = {}
 
-        # Checking a heuristic for determining if a build is a container build since, currently
+        # Checking heuristics for determining if a build is a container build since, currently
         # there is no definitive way to do it.
         if extra_json and (extra_json.get('container_koji_build_id') or
                            extra_json.get('container_koji_task_id')):
             return True
-        # Checking another heuristic for determining if a build is a container build since
-        # currently there is no definitive way to do it.
-        elif (package_name.endswith('-container') or package_name.endswith('-docker')):
+        elif extra_json.get('image') and (package_name.endswith('-container') or package_name.endswith('-docker')):
```

Comment:
flake8!

Reply:
oops! `ctrl+shift+R` didn't work it seems:p